recipes/ligo-common/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0.3" %}
+
+package:
+  name: ligo-common
+  version: {{version}}
+
+source:
+  fn: ligo-common-{{version}}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/ligo-common-{{version}}.tar.gz
+  md5: 142c2e287cc09bd0192c916a0be6e492
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - setuptools
```

Comment:
Why do we need `setuptools` at run time?

Reply:
@jakirkham, because this package provides a namespace package that requires `pkg_resources` to declare itself upon import. The contents of the `ligo/__init__.py` file are thus:

```python
# license
__import__('pkg_resources').declare_namespace(__name__)
```

Reply:
Thanks for clarifying.